zuchini-intellij-support/src/main/java/org/zuchini/intellij/ZuchiniMain.java

```diff
@@ -39,10 +40,18 @@ public static void main(String[] args) throws Throwable {
         }
         ClassLoader cl = Thread.currentThread().getContextClassLoader();
 
-        run(cl, featureFiles, glue, name);
+        // Exit the jvm to stop any running non-daemon threads, for example embedded servlet containers
+        // Use same exit codes as IntelliJ JUnit4IdeaTestRunner
+        // See https://github.com/JetBrains/intellij-community/blob/7fa96faf725f9ee605444bacb794c21906a3e699/plugins/junit_rt/src/com/intellij/junit4/JUnit4IdeaTestRunner.java#L69
+        try {
+            Result result = run(cl, featureFiles, glue, name);
+            System.exit(result.wasSuccessful() ? 0 : -1);
+        } catch (Throwable throwable) {
+            System.exit(-2);
```

Comment:
Print stacktrace?

Reply:
I believe you get the catch clause for free, i.e. you don't need to handle the exception.

Reply:
I can't remember which exit status you'd get, but it's not 0.